scripts/dts/gen_defines.py

```diff
@@ -118,6 +118,12 @@ def main():
                 out_dt_define(f"{node.z_path_id}_PARENT",
                               f"DT_{node.parent.z_path_id}")
 
+            out_comment(f"Node's node labels:")
+            out_dt_define(node.z_path_id + f"_NODELABEL_NUM", len(node.labels))
+            for idx, label in enumerate(node.labels):
+                out_dt_define(node.z_path_id + f"_NODELABEL_IDX_{idx}_EXISTS", 1)
+                out_dt_define(node.z_path_id + f"_NODELABEL_IDX_{idx}", label)
```

Comment:
Why using both string concatenation and f-string here ?

I suggest to purely use f-string.
```suggestion
                out_dt_define(f"{node.z_path_id}_NODELABEL_IDX_{idx}_EXISTS", 1)
                out_dt_define(f"{node.z_path_id}_NODELABEL_IDX_{idx}", label)
```